flexget/plugins/sites/jackett_torznab.py

```diff
@@ -0,0 +1,141 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # pylint: disable=unused-import, redefined-builtin
+from past.utils import old_div
+from future.moves.urllib.parse import urlencode, quote_plus
+
+import logging
+
+from flexget import plugin
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.utils.search import normalize_unicode
+import feedparser
+
+__author__ = 'davst'
+
+log = logging.getLogger('jackett_torznab')
+
+
+class Jackett_torznab(object):
+    """
+    Jackett_torznab search plugin
+    Based on the newznab plugin
+    Provide a url or your website + apikey and a category
+
+    Config example::
+
+        Jackett_Torznab:
+          url: "http://website/api?apikey=xxxxxxxxxxxxxxxxxxxxxxxxxx&t=movie&extended=1"
+          website: https://website
+          apikey: xxxxxxxxxxxxxxxxxxxxxxxxxx
+          category: movie
+
+    Category is any of: movie, tvsearch, search
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'category': {'type': 'string', 'enum': ['movie', 'tvsearch', 'tv', 'search']},
+            'url': {'type': 'string', 'format': 'url'},
+            'website': {'type': 'string', 'format': 'url'},
+            'apikey': {'type': 'string'}
+        },
+        'required': ['category'],
+        'additionalProperties': False
+    }
+
+    def build_config(self, config):
+        log.debug(type(config))
+
+        if config['category'] == 'tv':
+            config['category'] = 'tvsearch'
+
+        if 'url' not in config:
+            if 'apikey' in config and 'website' in config:
+                params = {
+                    't': config['category'],
+                    'apikey': config['apikey'],
+                    'extended': 1
+                }
+                config['url'] = config['website'] + '/api?' + urlencode(params)
+
+        return config
+
+    def fill_entries_for_url(self, url, task):
+        entries = []
+        log.verbose('Fetching %s' % url)
```

Comment:
Always let the logger do the string formatting ie. `log.verbose('Fetching %s', url)` (notice the comma).